src/pages/laatija.svelte

```diff
@@ -0,0 +1,118 @@
+<script>
+  import {
+    laatijat as laatijatStore,
+    patevyydet,
+    toimintaalueet
+  } from '@/stores';
+  import * as LaatijaUtils from '@/utilities/laatija';
+  import * as GeoUtils from '@/utilities/geo';
+  import { locale, labelLocale } from '@Localization/localization';
+
+  import Container, { styles as containerStyles } from '@Component/container';
+  import Button, { styles as buttonStyles } from '@Component/button';
+  import Spinner from '@Component/spinner';
+  import { onMount } from 'svelte';
+
+  export let id;
+  let component = null;
+
+  let laatijaPromise = Promise.all([
+    $laatijatStore,
+    $patevyydet,
+    $toimintaalueet
+  ])
+    .then(values => deserialize(values))
+    .then(laatijat => laatijat.find(laatija => laatija.id == id));
```

Comment:
Tän pitäis varmaan tuottaa rejektoitu promise jossei tuota löydy. Silloin käyttäjä lienee syöttänyt jotain ihan puppua urliin.